test/extended/networking.sh

```diff
@@ -7,7 +7,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-true}" != "true" ]]; then
```

Comment:
OPENSHIFT_CONFIG_ROOT should probably be considered as well, since ci mode (deploying clusters for each plugin) is only used when that env var is found to be empty.